tests/test_plugin_ccxt.py

```diff
@@ -396,12 +396,16 @@ def test_fiat_pair(self, mocker: Any) -> None:
     def test_kraken_csv(self, mocker: Any) -> None:
         plugin: PairConverterPlugin = PairConverterPlugin(Keyword.HISTORICAL_PRICE_HIGH.value, google_api_key="whatever")
 
-        cache_path = os.path.join(CACHE_DIR, plugin.cache_key())
+        cache_path = os.path.join(CACHE_DIR, "Test-" + plugin.cache_key())
         if os.path.exists(cache_path):
             os.remove(cache_path)
 
         kraken_csv = KrakenCsvPricing(google_api_key="whatever")
-        with open("input/test_kraken_CSV.zip", "rb") as file:
+        mocker.patch.object(kraken_csv, "cache_key").return_value = "Test-" + kraken_csv.cache_key()
+        mocker.patch.object(kraken_csv, "_Kraken__CACHE_DIRECTORY", "input/kraken_test")
+        if not os.path.exists("input/kraken_test"):
+            os.makedirs("input/kraken_test")
```

Comment:
Does this write in the top level input directory? Any test-generated files and directories should go into the output directory.

Reply:
Yeah, output-ed files should go in output. Go figure. Thanks. :)